test/integration/hapi-statsd.tests.js

```diff
@@ -33,36 +36,118 @@ beforeEach(function(done) {
 		reply(new Error());
 	};
 
-	server.route({ method: ['GET','OPTIONS'], path: '/', handler: get, config: {cors: true}});
+	var err407 = function (request, reply) {
+		reply('error').statusCode = 407;
+	};
+
+	server.route({ method: ['GET','OPTIONS'], path: '/', handler: get, config: {cors: true} });
 	server.route({ method: 'GET', path: '/err', handler: err, config: {cors: true} });
-	server.route({ method: 'GET', path: '/test/{param}', handler: get, config: {cors: true}});
+	server.route({ method: 'GET', path: '/test/{param}', handler: get, config: {cors: true} });
+	server.route({ method: 'GET', path: '/default', handler: get, config: {cors: true} });
+	server.route({ method: 'GET', path: '/override', handler: get, config: {cors: true} });
+	server.route({ method: 'GET', path: '/rename', handler: get, config: {cors: true} });
+	server.route({ method: 'GET', path: '/match/id', handler: get, config: {id: 'match-my-id', cors: true} });
+	server.route({ method: 'POST', path: '/match/method', handler: get, config: { cors: true} });
+	server.route({ method: 'GET', path: '/match/status', handler: err407, config: { cors: true} });
 
 	server.register({
 		register: plugin,
-		options: { statsdClient: mockStatsdClient }
+		options: {
+			statsdClient: mockStatsdClient,
+			defaultFilter: {
+				enableCounter: false,
+				enableTimer: true,
+			},
+			filters: [
```

Comment:
Just curious why you've set this up so that these filters are set at a plugin level as opposed to setting them directly on the route config? I think if you truly want to disable metrics for a certain method or statusCode, that can be set at the plugin level, but this seems like a weird spot for the route-specific ones.